file.c

```diff
@@ -5222,47 +5222,75 @@ rb_thread_flock(void *data)
     return (VALUE)ret;
 }
 
-/*
+/*  :markup: markdown
+ *
  *  call-seq:
- *     file.flock(locking_constant) -> 0 or false
- *
- *  Locks or unlocks a file according to <i>locking_constant</i> (a
- *  logical <em>or</em> of the values in the table below).
- *  Returns <code>false</code> if File::LOCK_NB is specified and the
- *  operation would otherwise have blocked. Not available on all
- *  platforms.
- *
- *  Locking constants (in class File):
- *
- *     LOCK_EX   | Exclusive lock. Only one process may hold an
- *               | exclusive lock for a given file at a time.
- *     ----------+------------------------------------------------
- *     LOCK_NB   | Don't block when locking. May be combined
- *               | with other lock options using logical or.
- *     ----------+------------------------------------------------
- *     LOCK_SH   | Shared lock. Multiple processes may each hold a
- *               | shared lock for a given file at the same time.
- *     ----------+------------------------------------------------
- *     LOCK_UN   | Unlock.
+ *    flock(locking_constant) -> 0 or false
+ *
+ *  Locks or unlocks a file according to the given `locking_constant`,
+ *  a bitwise OR of the values in the table below.
+ *
+ *  Not available on all platforms.
  *
+ *  Returns `false` if `File::LOCK_NB` is specified and the operation would have blocked;
+ *  otherwise returns `0`.
+ *  <br>
+ *
+ *  <table>
+ *  <tr>
+ *  <th colspan="3">Locking Constants</th>
+ *  </tr>
+ *  <tr>
+ *  <th>Constant</th>
+ *  <th>Lock</th>
+ *  <th>Effect</th>
+ *  </tr>
+ *  <tr>
+ *  <td><tt>File::LOCK_EX</tt></td>
+ *  <td>Exclusive</td>
+ *  <td>Only one process may hold an exclusive lock for <tt>self</tt> at a time.</td>
+ *  </tr>
+ *  <tr>
+ *  <td><tt>File::LOCK_NB</tt></td>
+ *  <td>Non-blocking</td>
+ *  <td>
+ *  No blocking; may be combined with other `File::LOCK_SH` or `File::LOCK_EX`
+ *  using the bitwise OR operator <tt>|</tt>.
+ *  </td>
+ *  </tr>
+ *  <tr>
+ *  <td><tt>File::LOCK_SH</tt></td>
+ *  <td>Shared</td>
+ *  <td>Multiple processes may each hold a shared lock for <tt>self</tt> at the same time.</td>
+ *  </tr>
+ *  <tr>
+ *  <td><tt>File::LOCK_UN</tt></td>
+ *  <td>None</td>
+ *  <td>No locking.</td>
+ *  </tr>
+ *  </table>
```

Comment:
We should indent the table so it's more readable.

```suggestion
 *  <table>
 *    <tr>
 *      <th colspan="3">Locking Constants</th>
 *    </tr>
 *    <tr>
 *      <th>Constant</th>
 *      <th>Lock</th>
 *      <th>Effect</th>
 *    </tr>
 *    <tr>
 *    <td><tt>File::LOCK_EX</tt></td>
 *    <td>Exclusive</td>
 *      <td>Only one process may hold an exclusive lock for <tt>self</tt> at a time.</td>
 *    </tr>
 *    <tr>
 *      <td><tt>File::LOCK_NB</tt></td>
 *      <td>Non-blocking</td>
 *      <td>
 *        No blocking; may be combined with other `File::LOCK_SH` or `File::LOCK_EX`
 *        using the bitwise OR operator <tt>|</tt>.
 *      </td>
 *    </tr>
 *    <tr>
 *      <td><tt>File::LOCK_SH</tt></td>
 *      <td>Shared</td>
 *      <td>Multiple processes may each hold a shared lock for <tt>self</tt> at the same time.</td>
 *    </tr>
 *    <tr>
 *      <td><tt>File::LOCK_UN</tt></td>
 *      <td>None</td>
 *      <td>No locking.</td>
 *    </tr>
 *  </table>
```